.github/workflows/generate-psi-report.yml

```diff
@@ -0,0 +1,29 @@
+name: Generate PSI report
+on:
+  # Run once a week on 01:30 every Wednesday
+  schedule:
+  - cron: "30 1 * * 3"
+
+jobs:
+  run_psi:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+         persist-credentials: false
+         fetch-depth: 0
+
+      - name: psi-gh-action
+        uses: mazipan/psi-gh-action@1.2.0
+        with:
+          api_key: ${{ secrets.PSI_API_KEY }}
+          urls: |
+            https://jefrydco.id/
+            https://jefrydco.id/blog
+          devices: |
+            mobile
+            desktop
+          runs: 1
+          branch: master
```

Comment:
Hmm, the Action isn't triggered, I think this config should be `main`. Let me try to change this Mas

Reply:
It's not triggered by push, it's by the Cron schedule.
It's better to have a stable schedule rather than using push trigger.
But for testing purpose, it's okey to set using push action first.

Reply:
For safety, add a path ignore when using push event

```yaml
on:
  push:
    branches:
    - main
    paths-ignore:
    - "psi-reports/**"
```

Reply:
Seems the page in https://jefrydco.id/speed still 404.
Is there anything wrong?
Is the auto deployment broken?

Just realize that your blog didn't handle 404 error properly 😂
Still fallback to default from Netlify.

Reply:
Ah, I think I miss to add /speed to the generate function.
So it's not generate the html file.

Reply:
Umm, actually I disabled auto deployment in Netlify since its yield weird behaviour, it makes the live demo on my blog didn't work. Currently I deployed it locally using Netlify CLI. :smile: 

I just added new GitHub Actions on this PR #119 

> Just realize that your blog didn't handle 404 error properly
I don't know that we can customize the 404 error, but I'll get into that on #119 as well.

> For safety, add a path ignore when using push event
Sure Mas, I added that snippet

Reply:
It should be no need to add `/speed` to generate function since its already declared as a page. It just for dynamically rendered path.

Reply:
Btw Mas @irfan-maulana-tkp , it seems the result of the psi test is always returns 0, is there anything wrong with my config?
https://github.com/jefrydco/jefrydco/runs/1885520069?check_suite_focus=true

Reply:
Do you have set API key?